tests/ct_pipeline.c

```diff
@@ -0,0 +1,136 @@
+#include "adapters/libevent.h"
+#include "hircluster.h"
+
+#include <assert.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define CLUSTER_NODE "127.0.0.1:30001"
+
+// Test of two pipelines using sync API
+void test_pipeline() {
+    redisClusterContext *cc = redisClusterContextInit();
+    assert(cc);
+    redisClusterSetOptionAddNodes(cc, CLUSTER_NODE);
+    redisClusterConnect2(cc);
+
+    assert(cc->err == 0);
+
+    int status;
+    status = redisClusterAppendCommand(cc, "SET foo 5");
```

Comment:
Here 5 is a string because it is sent as the non-binary-safe space-separated command syntax, right? If we want to send 5 as an integer, I guess "SET %s %d" should be used, with that formatting function. Perhaps it doesn't matter for the purpose of this test case, but I think sending "5" as a string is slightly misleading for anyone who uses the test case as a usage example.

Reply:
you are right, i will change to a less misleading string